keps/sig-api-machinery/3352-aggregated-discovery/README.md

```diff
@@ -620,7 +620,11 @@ main focus will be on kubectl and golang clients.
 
 #### GA
 
-- TBD
+- Existing bugs are fixed:
+  - AggregatedDiscovery controller does not purge old APIServices from cache ([Issue](https://github.com/kubernetes/kubernetes/issues/115301))
+  - Aggregated Discovery doesn't show aggregated apiservices as Stale before initial health check ([Issue](https://github.com/kubernetes/kubernetes/issues/115303))
+- New API type `apidiscovery.k8s.io/v2` is introduced
```

Comment:
Should the `Design Details` section use the v2 api, not the v2beta1? With a pointer to `Version Skew` laying out details about handing both of these?

Reply:
Updated